CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -10,14 +10,16 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 - Fix [server/embed: fix data race when starting both secure & insecure gRPC servers on the same address](https://github.com/etcd-io/etcd/pull/15518)
 - Add [`etcd --tls-min-version --tls-max-version`](https://github.com/etcd-io/etcd/pull/15486) to enable support for TLS 1.3.
 - Fix [server/auth: disallow creating empty permission ranges](https://github.com/etcd-io/etcd/pull/15621)
-
-### Go
-- Require [Go 1.19+](https://github.com/etcd-io/etcd/pull/15333).
-- Compile with [Go 1.19+](https://go.dev/doc/devel/release#go1.19)
+- Upgrade [golang.org/x/net to v0.7.0](https://github.com/etcd-io/etcd/pull/15333).
+- Add [`etcd --listen-client-http-urls`](https://github.com/etcd-io/etcd/pull/15620) flag to support separating http server from grpc one, thus giving full immunity to [watch stream starvation under high read load](https://github.com/etcd-io/etcd/issues/15402).
 
 ### Package `clientv3`
 - Reverted the fix to [auth invalid token and old revision errors in watch](https://github.com/etcd-io/etcd/pull/15542).
 
+### Go
+- Require [Go 1.19+](https://github.com/etcd-io/etcd/pull/15337).
```

Comment:
Noticed one thing. Isn't changing required minimal Go version in package a breaking change? We are basically forcing all users of `clientv3` to upgrade Go in their applications even though we didn't do needed to. 

I think that upgrading Go version used to compile etcdserver and required minimal Go version should be independent. We should also avoid changing the second one if not needed.

Reply:
cc @ahrtr @ptabor WDYT?

Reply:
It's recommended to use golang 1.19+ to build applications depending on clientv3, but it isn't a must. So suggest to change it to something like "`Recommend Go 1.19+`"

Reply:
Ok, https://go.dev/ref/mod#go-mod-file-go sais `If an older Go version builds one of the module’s packages and encounters a compile error, the error notes that the module was written for a newer Go version.`. Confirming it is more of a recommendation.